modules/installation-configuration-parameters.adoc

```diff
@@ -1586,6 +1590,48 @@ In {product-title} 4.12 and later, the `apiVIP` configuration setting is depreca
 In {product-title} 4.12 and later, the `ingressVIP` configuration setting is deprecated. Instead, use a list format to enter a value in the `ingressVIPs` configuration setting.
 ====
 
+ifdef::vsphere-zones[] 
+|`platform.vsphere.failureDomains`
+|Optional. The list of failure domain definitions.
+|Array of `failureDomain` definitions.
+
+|`platform.vsphere.failureDomains.*.name`
+|Required. The name of the failure domain. This name will be referred to by the machine pools.
```

Comment:
```suggestion
|Required. The name of the failure domain. The machine pools will use this name to reference the failure domain.
```
reworded to avoid passive voice